ui/components/app/gas-details-item/gas-details-item.js

```diff
@@ -64,20 +87,29 @@ const GasDetailsItem = ({
     hexWEIToDecGWEI(transactionData.txParams?.maxFeePerGas ?? '0x0')
   ).toString();
 
+  const getTransactionFeeTotal = () => {
+    if (isMultiLayerFeeNetwork) {
+      return sumHexes(hexMaximumTransactionFee, estimatedL1Fees || 0);
+    }
+
+    return hexMaximumTransactionFee;
+  };
+
```

Comment:
We can use `useMemo` to avoid performing expensive calculations
```suggestion
  const getTransactionFeeTotal = useMemo(() => {
    if (isMultiLayerFeeNetwork) {
      return sumHexes(hexMaximumTransactionFee, estimatedL1Fees || 0);
    }

    return hexMaximumTransactionFee;
}, [isMultiLayerFeeNetwork, hexMaximumTransactionFee, estimatedL1Fees]);
```